### ISSUE TITLE
snapshot contents shown on stackage.org not matching reality ?

### ISSUE THREAD

**Issue by simonmichael** [fpco/stackage-server#184]
As just discussed on #haskell-stack. Is stackage.org confused, or is it me ? Eg: https://www.stackage.org/package/hasktags says "This package is not currently in any snapshots.", but of course it is:
```shell
~$ stack install hasktags
Run from outside a project, using implicit global project config
Using resolver: lts-6.0 from implicit global project's config file: /Users/simon/.stack/global/stack.yaml
hasktags-0.69.1: configure
hasktags-0.69.1: build
hasktags-0.69.1: copy/register
...
```

**Comment by bergmark** [comment]
I think hasktags is not in stackage but stack resolves the dependencies with the snapshot and builds it anyway.

**Comment by simonmichael** [comment]
Right you are! I searched the user guide but failed to find this the first time. Now I see it:

http://docs.haskellstack.org/en/stable/GUIDE/#targets-locals-and-extra-deps:
*whether it's a local package, in your extra-deps, in your snapshot, or just available upstream*

and more clearly (today I noticed the build command has an additional guide all of its own!):

http://docs.haskellstack.org/en/stable/build_command/#target-syntax:
*It will try to find the package in the following locations: local packages, extra dependencies, snapshots, and package index (e.g. Hackage).*

stack does so much it's hard to keep track of it all. Thanks.
